iosApp/iosApp/Pages/StopDetails/StopDetailsFilteredView.swift

```diff
@@ -17,12 +17,16 @@ struct StopDetailsFilteredView: View {
     var setStopFilter: (StopDetailsFilter?) -> Void
     var setTripFilter: (TripDetailsFilter?) -> Void
 
-    var departures: StopDetailsDepartures?
+    var routeCardData: [RouteCardData]?
     var now: Date
 
+    var routeData: RouteCardData?
+    var stopData: RouteCardData.RouteStopData?
+    var leafData: RouteCardData.Leaf?
```

Comment:
If there’s nothing we’d show if, say, the `routeData` and `stopData` exist but the `leafData` is `nil`, would it make sense to just store one `DepartureDataBundle?`?